tutorials/gcs-cf-auth/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Google Cloud Functions Access Control
+description: Learn how to add authentication and authorization to Google Cloud Functions using Oauth tokens.
+author: ianmaddox
+tags: serverless, cloud functions, javascript, iam, authn, authz, authorization, authentication
+date_published: 2019-01-08
+---
+# Google Cloud Functions Access Control
```

Comment:
When I do the production pass, I'll be inserting blank lines between headings and body text to make our Markdown processor happy. You don't need to worry about this now, but it might be good to know for next time.